src/Commands/pm/SecurityUpdateCommands.php

```diff
@@ -177,7 +177,7 @@ public function securityPhp($options = ['format' => 'yaml'])
         $process->mustRun();
         $out = $process->getOutput();
         if ($packages = json_decode($out, true)) {
-            $suggested_command = "composer why " . key($packages);
+            $suggested_command = "composer why " . implode('; composer why ', array_keys($packages));
```

Comment:
```suggestion
            $suggested_command = "composer why " . implode('&& composer why ', array_keys($packages));
```
I think this is more Windows friendly.